plugin-flex-ts-template-v2/src/feature-library/keyboard-shortcuts/custom-components/KeyboardShortcuts/Tabs/DefaultKeyboardShortcutsView.tsx

```diff
@@ -0,0 +1,172 @@
+import { getDefaultShortcuts } from '../../../utils/KeyboardShortcutsUtil';
+
+interface DefaultKeyboardShortcutsViewProps {
+  reset: boolean;
+  disableShortcuts: boolean;
+  isThrottleEnabled: boolean;
+  isDeleteShortcutsEnabled: boolean;
+  toasterDeleteNotification: (actionName: string) => void;
+  toasterSuccessNotification: (actionName: string, oldShortcut: string, newShortcut: string) => void;
+}
+
+const DefaultKeyboardShortcutsView = ({
+  reset,
+  disableShortcuts,
+  isThrottleEnabled,
+  isDeleteShortcutsEnabled,
+  toasterDeleteNotification,
+  toasterSuccessNotification,
+}: DefaultKeyboardShortcutsViewProps) => {
+  const [defaultShortcuts, setDefaultShortcuts] = useState<ShortcutsObject[]>([]);
+  const [shortcutsDeleted, setShortcutsDeleted] = useState<boolean>(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState<boolean>(false);
+  const [selectedShortcutKey, setSelectedShortcutKey] = useState<string>('');
+  const [selectedActionName, setSelectedActionName] = useState<string>('');
+  const [selectedThrottle, setSelectedThrottle] = useState<number | undefined>(0);
+
+  const openModalHandler = (shortcut: string, actionName: string, throttle?: number): void => {
+    setIsEditModalOpen(true);
+    setSelectedThrottle(throttle);
+    setSelectedShortcutKey(shortcut);
+    setSelectedActionName(actionName);
+  };
+
+  const closeModalHandler = (): void => {
+    setIsEditModalOpen(false);
+  };
+
+  useEffect(() => {
+    setDefaultShortcuts(getDefaultShortcuts());
+    setShortcutsDeleted(false);
+  }, [reset]);
+
+  useEffect(() => {
+    if (Object.keys(getDefaultShortcuts()).length === 0) {
+      setShortcutsDeleted(true);
+    }
+  }, [defaultShortcuts]);
+
+  return (
+    <>
+      {/* Check if shortcuts have been disabled or deleted  */}
+      {disableShortcuts || shortcutsDeleted ? (
+        <Card>
+          <Heading as="h5" variant="heading50">
+            <Stack orientation="horizontal" spacing="space20">
+              <WarningIcon decorative={false} title="Description of icon" />
+              Re-enable Keyboard Shortcuts
+            </Stack>
+          </Heading>
+          <Paragraph>
+            There are no configured keyboard shortcuts. Please reset your keyboard settings to enable keyboard shortcut
+            customization.
+          </Paragraph>
+        </Card>
+      ) : (
+        <>
+          {/* Present a table if there are shortcuts to present */}
+          <Box overflowY="auto" maxHeight="1000px">
+            <Table>
+              <THead>
+                <Tr>
+                  <Th>
+                    <Stack orientation="horizontal" spacing="space30">
+                      <Tooltip text="Ctrl and Shift are the default modifiers that cannot be changed.">
+                        <Text as="span">Modifiers</Text>
+                      </Tooltip>
+                      <InformationIcon decorative={false} title="modifiers" />
```

Comment:
Should be made consistent with the Custom tab.

Reply:
Good catch. I've made it consistent.